definition_validator/validator.cc

```diff
@@ -179,7 +179,7 @@ void validateOverriding(const core::GlobalState &gs, core::SymbolRef method) {
             }
         }
         if ((overridenMethod.data(gs)->isAbstract() || overridenMethod.data(gs)->isOverridable()) &&
-            !method.data(gs)->isIncompatibleOverride()) {
+            !method.data(gs)->isIncompatibleOverride() && !method.data(gs)->loc().file().data(gs).isRBI()) {
```

Comment:
I think we might want to iterate over all `locs()`

Reply:
Which are `locs()` are going to be important here? I think the one that's associated with the function head should probably be sufficient to accept or reject, but maybe I'm not thinking of a specific case.

Reply:
Try the following test cases:

Test case 1:
```
# foo.rbi
class S
  include Enumerable
  sig { void }
  def each; end
end
```

```
# foo.rb
class S
  def each; end
end
```

test case 2:

```
# foo.rbi
class S
  include Enumerable
  def each; end
end
```

```
# foo.rb
class S
  sig { void }
  def each; end
end
```

test case 3:

```
# foo.rbi
class S
  include Enumerable
  def each; end
end
```

```
# foo.rb
class S
  def each; end
end
```

test case 4:

```
# foo.rbi
class S
  include Enumerable
  sig { void }
  def each; end
end
```

```
# foo.rb
class S
  sig { void }
  def each; end
end
```

Reply:
You're right, I had been forgetting situations like these. This has been fixed.